components/collapsible/collapsible.mdx

```diff
@@ -58,10 +60,40 @@ import { DtCollapsible, DtButton } from '@dialpad/dialtone-vue';
       Click Me!
     </dt-button>
   </template>
-  <template #content>
+  <template #contentOnExpanded>
     <p>
       This will be shown in the expanded area.
     </p>
   </template>
 </dt-collapsible>
 ```
+
+### Basic Usage with `contentOnCollapsed`
+
+You can also choose to show content when under the anchor when it is collapsed. To do so,
```

Comment:
remove 'when'

"You can also choose to show content under the anchor when it is collapsed"